app/controllers/integrations/base_controller.rb

```diff
@@ -16,7 +16,8 @@ def create
 
     if branch
       create_release = project.create_release?(branch, service_type, service_name)
-      record_log :info, "Branch #{branch} is release branch: #{create_release}"
+      record_log :info, "Create release for "\
+        "branch [#{branch}], service_type [#{service_type}], service_name [#{service_name}]: #{create_release}"
```

Comment:
```suggestion
      record_log(
        :info, 
        "Created release for branch: #{branch}, service_type: #{service_type}, service_name: #{service_name}: #{create_release}"
      )
```

Reply:
It's still a _little_ too long for Rubocop's liking :(

Reply:
dammit :D